crates/core/src/kernel/models/actions.rs

```diff
@@ -377,6 +377,97 @@ impl Protocol {
     }
 }
 
+/// High level table features
+#[derive(Serialize, Deserialize, Debug, Clone, Eq, PartialEq, Hash)]
+#[serde(rename_all = "camelCase")]
+pub enum TableFeatures {
```

Comment:
What should the relation between this enum and the `DeltaConfigKey` in `src/table/config.rs` be? :thinking: :confused: